src/wasm/cpp/tests/wasm-cpp-test.ts

```diff
@@ -268,4 +278,112 @@ describe('wasm tests (C++)', () => {
       assert.fail(`${errors.length} errors found:\n${errors.join('\n')}`);
     }
   });
+
+  prefix('reference class API', async () => {
+    const {stores} = await setup(`
+      import '${schemasFile}'
+
+      particle ReferenceClassApiTest in '${buildDir}/test-module.wasm'
+        out [Data] errors
+
+      recipe
+        ReferenceClassApiTest
+          errors -> h1
+      `);
+    const errStore = stores.get('errors') as VolatileCollection;
+    const errors = (await errStore.toList()).map(e => e.rawData.txt);
+    if (errors.length > 0) {
+      assert.fail(`${errors.length} errors found:\n${errors.join('\n')}`);
+    }
+  });
+
+  it('reading from reference-typed handles', async () => {
```

Comment:
did you want all the test cases to use `prefix` instead of `it`, or just the two above?

Reply:
Nope, just the ones I know print out test case names. The `prefix` thing causes the main name to be printed twice (because mocha prints test names after any logging the test outputs).